ehitchhiking-ui/src/styles/modals.sass

```diff
@@ -1,2 +1,52 @@
+@import 'fonts-constants'
+@import 'colors'
+@import 'constants'
+@import 'mixins'
+
+.modal-wrapper
+  overflow: hidden
+  position: relative
+  .modal-header
+    width: 100%
+    height: 4rem
+    font-size: $headline
+    @include flexbox()
+    background-color: mat-color($accent, 300)
+    color: $white
+  .modal-content
+    margin: $default-offset
+  .modal-footer
+    display: flex
+    align-items: center
+    justify-content: center
+    margin: 0 $default-offset $default-offset $default-offset
+    .modal-action-btn
+      background-color: mat-color($primary, 50)
+  .modal-close
+    position: absolute
+    top: 0.5rem
+    right: 0.5rem
+    @include flexbox()
+    color: mat-color($primary, 400)
+    &-btn:hover
+      background-color: mat-color($accent, 50)
+
+::-webkit-scrollbar
```

Comment:
all scroll styles move to separate file or styles.sass

Reply:
moved to styless.sass